core/domain/state_domain.py

```diff
@@ -3257,6 +3257,16 @@ def _get_all_translatable_content(self):
 
         return content_id_to_translatable_item
 
+    def get_all_translatable_content(self):
```

Comment:
Hm. Why not just have a single public method then? Doesn't make sense to have a public and private method for exactly the same thing.

Or, perhaps better, use one of the existing public methods (that call this one internally)?

Reply:
Done